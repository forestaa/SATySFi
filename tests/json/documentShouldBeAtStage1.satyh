@stage: 0
let x = 3
