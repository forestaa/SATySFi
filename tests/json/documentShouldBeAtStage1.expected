{
  "file": "",
  "range": {
    "start": { "line": 0, "character": 0 },
    "end": { "line": 0, "character": 0 }
  },
  "message":
    "! [Error] invalid stage designation for a document file; should be at stage 1."
}
